index.html

```diff
@@ -79,12 +79,23 @@ <h2>CapturedMouseEvent interface</h2>
       <pre class="idl">
         [Exposed=Window]
         interface CapturedMouseEvent : Event {
-          constructor(optional CapturedMouseEventInit eventInitDict = {});
+          constructor(DOMString type, optional CapturedMouseEventInit eventInitDict = {});
           readonly attribute long surfaceX;
           readonly attribute long surfaceY;
         };
       </pre>
       <dl data-link-for="CapturedMouseEvent" data-dfn-for="CapturedMouseEvent">
+        <dt><dfn constructor>constructor()</dfn></dt>
+        <dd>
+          <p>Constructs a new {{CapturedMouseEvent}}.</p>
+          <p>The arguments are passed as-it to {{Event}}'s constructor.</p>
+          <p>If an {{CapturedMouseEventInit}} argument is provided then the
+            constructor <a data-cite="WEBIDL#dfn-throw">throws</a> a
+            <a data-cite="WEBIDL#exceptiondef-rangeerror">RangeError</a>
+            exception if any of {{CapturedMouseEventInit.surfaceX}} or
+            {{CapturedMouseEventInit.surfaceY}} is negative, and they are not
+            both equal to -1.
```

Comment:
Instead of `if x1 then y provided that also x2`, could you phrase this as `if x1 and x2 then y`?

Given that there's a default value, btw, how can no `eventInitDict` be provided? Would this happen if the argument is `null` or `undefined` or...? (IIANM the former is possible, but not the second.)

Reply:
Right. `eventInitDict` has default value `{}` and `CapturedMouseEventInit`'s `surfaceX`/`surfaceY` have default value -1. So IIUC the only outcome is that a `TypeError` for the second argument or `surfaceX`/`surfaceY` are well-defined (defaulting to -1).

Quick test for `Event`: (https://dom.spec.whatwg.org/#event)

```
function log_event(event) {
  console.log(`bubbles: ${event.bubbles}, cancelable: ${event.cancelable}, ${event.composed}`);
}

log_event(new Event("custom"));
log_event(new Event("custom", undefined));
log_event(new Event("custom", null));
log_event(new Event("custom", {}));
log_event(new Event("custom", {bubbles: true}));
log_event(new Event("custom", 7));
```